packages/itwinui-css/src/alert/alert.scss

```diff
@@ -9,35 +9,35 @@
 
   --_iui-alert-border-color: var(--iui-color-border);
 
-  border-radius: var(--iui-border-radius-1);
+  border-radius: var(--_iui-alert-border-radius, var(--iui-border-radius-1));
   display: flex;
   align-items: center;
   color: var(--iui-color-text);
   background-color: var(--iui-color-background);
   border: 1px solid var(--_iui-alert-border-color);
   box-shadow: inset var(--iui-size-2xs) 0 0 var(--_iui-alert-border-color);
-  @media (forced-colors: active) {
```

Comment:
Do we not need forced colors anymore?

Reply:
`@media (forced-colors: active) { background-color: Canvas; }` was needed when we [set the background color using this gradient hack](https://github.com/iTwin/iTwinUI/blob/3b9b327ed8cf84db87dc1d61d356445cd5b051ba/packages/itwinui-css/src/alert/alert.scss#L58).  Now that we use `background-color` the `background-color: Canvas` isn't necessary. 
![Screenshot 2022-11-09 at 2 56 59 PM](https://user-images.githubusercontent.com/849817/200928556-7d6499a3-a01b-44ef-b447-9e82ba6b713d.png)